buildpipeline/DotNet-CoreFx-Trusted-OSX.json

```diff
@@ -241,6 +264,29 @@
         "Parallel": "false",
         "ParallelCount": "8"
       }
+    },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Run end.sh",
+      "timeoutInMinutes": 0,
+      "condition": "succeeded()",
```

Comment:
the ps scripts are set to run `succeededOrFailed` which I think is a good idea for these as well.